H4LCppModule.py

```diff
@@ -0,0 +1,181 @@
+from PhysicsTools.NanoAODTools.postprocessing.framework.eventloop import Module
+import ROOT
+import os
+ROOT.PyConfig.IgnoreCommandLineOptions = True
+
+
+# MHT producer, unclean jets only (no lepton overlap cleaning, no jet selection)
+class HZZAnalysisCppProducer(Module):
+    def __init__(self):
+        if "/H4LTools_cc.so" not in ROOT.gSystem.GetLibraries():
+            print("Load C++ module")
+            base = "/afs/cern.ch/user/y/yujil/Run3H4l/CMSSW_10_6_20/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/nanoAOD_vvVBS" 
```

Comment:
Don't add the absolute path. Add the relative path. Use the environment variable `$CMSSW_BASE`